### DISCUSSION TITLE
support browser bio unlock

### DISCUSSION THREAD

**Discussion by creatorKoo** [dani-garcia/vaultwarden#1531] [Q&A] [answered]
I using bitwarden_rs and mac appstore version bitwarden desktop and firefox(87.0) browser extension
everything up-to-date version for now.

But, I can't activate browser extension. maybe bitwarden_rs server not support it?

Please, support browser bio unlock.

**Comment by jjlin** [comment, marked as answer]
Biometric unlock should be purely a client-side thing, and I've used it on Windows before.

There appear to be various issues with Firefox biometric unlock, on Mac at least (not sure if it's specific to Mac). There are some potential workarounds discussed at

https://community.bitwarden.com/t/when-will-the-firefox-extension-get-biometric-support-now-that-i-think-it-supports-nativemessaging/17354/18

**Reply by creatorKoo** [reply]
wow! that's perfectly work! 
thank you @jjlin :)
